AI/Academy/1-Intro-To-ML-And-MLOps/README.md

```diff
@@ -0,0 +1,98 @@
+# Lab: Modello ML di base con Dataset Meteo + Integrazione MLflow
+
+Benvenuto in questo laboratorio! Qui imparerai a:
+
+1. **Caricare e preparare un dataset meteo**, con dati di temperatura e umidità.
+2. **Addestrare un modello di Machine Learning** con Scikit-learn per prevedere la pioggia.
+3. **Valutare il modello** e comprendere i risultati.
+4. **Integrare MLflow** per tracciare metriche, parametri e versioni del modello.
+
+Seguiremo un approccio guidato, con spiegazioni dettagliate ad ogni passaggio.  
+La prima parte si concentra su Scikit-learn e il dataset meteo. La seconda parte estende il codice esistente con MLflow.
+
+---
+
+## Parte 0: Setup dell'Environment per Python
+
+Prima di iniziare, dobbiamo assicurarci di avere tutto il necessario per eseguire correttamente il laboratorio.  
+
+#### **Requisiti**
+
+Questo laboratorio assume che **Python** e **miniconda** siano già installati, la repository [kiratech/labs](https://github.com/kiratech/labs.git) sia accessibile e che Git sia configurato correttamente sul computer locale.
+
+### 1. Clonare la Repository
+Per iniziare, clona la repository del laboratorio eseguendo il seguente comando nel terminale:
+```sh
+  git clone https://github.com/kiratech/labs.git
+```
+
+### 2. Fare il Checkout del Branch Lab
+
+Dopo aver clonato la repository, entra nella cartella del progetto:
+
+```sh
+  cd labs
+```
+
+Quindi, fai il checkout del branch `lab`:
+
+```sh
+  git checkout academy-ai
+```
+In questa cartella sono presenti le risorse relative ai laboratori con tema AI.
+
+### 3. Spostarsi nella Cartella 1-Intro-To-ML-And-MLOps
+
+Naviga nella cartella del primo laboratorio:
+
+```sh
+  cd AI/Academy/1-Intro-To-ML-And-MLOps
+```
+
+### 4. Apri il progetto in VSCode
+
+A questo punto apri VSCode dall'explorer oppure lanciando il comando:
+```sh
+  code .
+```
+
+### 5. Creare un Virtual Environment
+
+Un ambiente virtuale permette di isolare le dipendenze del progetto da quelle di sistema.
+
+Tramite il terminale di **VSCode**, crea un ambiente virtuale:
```

Comment:
VSCode is not presented as a requirement, but in fact, it is, so I think it's better to add it at the beginning.